### PR TITLE
Ed/jobs cleaner

### PR DESCRIPTION
**Pull request by EdwinRozario**
CLEAN UP OLD SIDEKIQ JOBS FROM MONGO

Acceptance Criteria
=================
As a developer i want the jobs that are processed to be deleted from Mongo.
Add timestamp fields to these Jobs if it dosent exist.
Create a cleaner job.
Run the cleaner job once in a week.

Background
=================
Mongo entries created for Sidekiq Jobs were never cleared. The Story aims at clearing these jobs and keeping only a few based on business decisions.

Checklist
=================

- [ ] Code is understandable without Dev
- [ ] Acceptance criteria, what was changed, and why it was changed (If applicable) on Pull / Merge Request
- [ ] All Tests are Passing
- [ ] Code Coverage goes up
- [ ] All new methods have a relevant spec
- [ ] Methods have a single responsibility (Where applicable)
- [ ] Linters Pass
- [ ] ‘Yard’ style comments on methods and classes (Where applicable)

---
This change is [Reviewable](https://reviewable.io/reviews/digitalnz/supplejack_worker/11)

### CODE REVIEW

**Comment by UberMouse** [comment]
👍

File: lib/tasks/sidekiq_jobs.rake [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# See https://github.com/DigitalNZ/supplejack_worker for details.
+#
+# Supplejack was created by DigitalNZ at the National Library of NZ
+# and the Department of Internal Affairs. http://digitalnz.org/supplejack
+
+namespace :sidekiq_jobs do
+  desc 'Delets old jobs from Mongo'
+  task purge: :environment do
+    # Keeping jobs for last 500 days. These are Harvest Jobs
+    AbstractJob.where(:updated_at.lte => (Date.today - 500)).delete_all
```
Comment (DanHenton) [inline lib/tasks/sidekiq_jobs.rake]: why 500 days?

**Comment by DanHenton** [comment]
👍

File: lib/tasks/sidekiq_jobs.rake (hunk 1 above)
Comment (EdwinRozario) [inline lib/tasks/sidekiq_jobs.rake]: Business decision.

**Pull request merged**